webapp/src/components/cardDetail/comment.tsx

```diff
@@ -60,6 +61,21 @@ const Comment: FC<Props> = (props: Props) => {
                         />
                     </Menu>
                 </MenuWrapper>
+                <div className='comment-date'>{(new Date(comment.createAt)).toLocaleString()}</div>
```

Comment:
This looks like some kind of accidentally included duplication.

Reply:
Yeah, one component was duplicated. Updating now.

Reply:
Donel